app/containers/EliteApiLoader/reducer.js

```diff
@@ -178,12 +179,11 @@ function EliteApiReducer(state = initialState, action) {
       return state.setIn(['User', 'CaseLoads'], fromJS({ Status: { Type: 'ERROR', Error: action.payload.error } }))
     }
 
-    case USER.ROLES.SUCCESS: {
```

Comment:
can't see this code being used anywhere and certainly not being set on the state